messaging/internal/service/channel.go

```diff
@@ -54,6 +55,9 @@ type (
 		Delete(ID uint64) (*types.Channel, error)
 		Undelete(ID uint64) (*types.Channel, error)
 		RecordView(userID, channelID, lastMessageID uint64) error
+
+		WebhookList(channelID uint64) (types.WebhookSet, error)
```

Comment:
let's reverse this --- channels should not be aware of webhooks, webhooks are always (?) aware of channels.